client/src/containers/Search/Search.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import './Search.css';
+import axios from 'axios';
+import MovieList from '../../components/MovieList/MovieList';
+import { Row } from 'react-bootstrap';
+
+const Search = (props) => {
+
+  const [ movies , setMovies ] = useState([]);
+  const [ query , setQuery ] = useState('');
+  const [ timeOut, setTimeOut ] = useState(0);
+  const baseUrl = 'http://localhost:3001';
```

Comment:
This baseUrl for the local host has been defined in three different files. Maybe there's a more central location it can be stored once for reference.